circus-train-housekeeping/src/main/java/com/hotels/bdp/circustrain/housekeeping/CircusTrainHousekeepingConfiguration.java

```diff
@@ -39,8 +40,10 @@
   @Bean
   HousekeepingService housekeepingService(
       LegacyReplicaPathRepository legacyReplicaPathRepository,
-      @Qualifier("baseConf") org.apache.hadoop.conf.Configuration baseConf) {
-    return new FileSystemHousekeepingService(legacyReplicaPathRepository, baseConf);
+      @Qualifier("baseConf") org.apache.hadoop.conf.Configuration baseConf,
+      Housekeeping housekeeping) {
+    return new FileSystemHousekeepingService(legacyReplicaPathRepository, baseConf,
```

Comment:
Doesn't this require a bump in the version of HK that CT depends on?

Reply:
yes the PR is on a branch which already bumps it, I'll merge the other one in first as soon as HK is released

Reply:
OK, makes sense.